src/component/metric/index.tsx

```diff
@@ -5,6 +5,7 @@ import {
   PlasmicMetric,
   DefaultMetricProps
 } from "../plasmic/flip_card/PlasmicMetric";
+import LineChart from '../linechart/index'
```

Comment:
We should be able to do something like this. 

```suggestion
import * as linechart from 'component/linechart'
```

Reply:
ok